src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/UnprotectedIncentiveReviewsResourceTestLevel.kt

```diff
@@ -1,173 +0,0 @@
-package uk.gov.justice.digital.hmpps.incentivesapi.resource
```

Comment:
Would be better to only remove the tests for the specific endpoints that were removed instead of the whole test file for the resource?

Reply:
The endpoint is just a wrapper, doesn't seem worth keeping it as the final endpoint will go shortly